docs/services/MicrosoftGraph.md

```diff
@@ -32,18 +32,22 @@ If you don't have one, you need to create an Office 365 Developer Site. There ar
  
 ### 2. Register you application in Azure Active Directory
 
-To authenticate your app, you need to register your app with Azure AD, and provide some details about your app. You can register your app manually by using the [Azure Management Portal](http://manage.windowsazure.com), or by using Visual Studio.
+To authenticate your app, you need to register your app with Azure AD, and provide some details about your app. 
+
+#### Register the App to use Azure AD v1 Endpoint
+
+You can register your app manually by using the [Azure Management Portal](http://portal.azure.com), or by using Visual Studio.
 
 To register your app manually, see [Manually register your app with Azure AD so it can access Office 365 APIs.](https://msdn.microsoft.com/en-us/office/office365/howto/add-common-consent-manually)
 
 To register your app by using Visual Studio, see [Using Visual Studio to register your app and add Office 365 APIs.](https://msdn.microsoft.com/office/office365/HowTo/adding-service-to-your-Visual-Studio-project)
 
 After you've registered your app, Azure AD will generate a client ID for your app. You'll need to use this client ID to get your access token.
 
-When you register your app in the [Azure Management Portal](http://manage.windowsazure.com), you will need to configure details about your application with the following steps:
+When you register your app in the [Azure Management Portal](http://portal.azure.com), you will need to configure details about your application with the following steps:
```

Comment:
Should we be more explicit about what they click on to register their app?  Like below for the v2 endpoint we say click on "Add an app"

Reply:
ping @luismanez, could you address these two comments so we can merge this PR?

Reply:
ops, sorry, yeah, will update the PR during today/tomorrow. thanks

Reply:
@nmetulev docs updated as per comments. Please, take a look and let me know if it needs any other change. Thanks!